src/config.py

```diff
@@ -41,6 +41,9 @@
 # Energy points each creature has at the beginning of the game
 CREATURE_STARTING_ENERGY: int = 30
 
+# Maximum creature energy 
```

Comment:
This is technically not the maximum amount, but a coefficient used with the creature size and starting energy. I think the comment here (which serves as a config documentation) should be a bit clearer